opentech/users/migrations/0002_initial_data.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.7 on 2017-12-15 13:15
+from __future__ import unicode_literals
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management.sql import emit_post_migrate_signal
+from django.db import migrations
+
+from opentech.users.groups import GROUPS
+
+
+def add_groups(apps, schema_editor):
+    # Workaround for https://code.djangoproject.com/ticket/23422
+    db_alias = schema_editor.connection.alias
+    try:
+        emit_post_migrate_signal(2, False, db_alias)
+    except TypeError:
```

Comment:
We have control over the django version and will not be targeting django < 1.11 so dont need this code.

Reply:
👍